### PR TITLE
Install or configure things related to OSCAP only if it's Sat6.1

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

**Comment by kbidarkar** [comment]
Fixes, https://github.com/SatelliteQE/automation-tools/issues/267

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1144,8 +1144,9 @@ def product_install(distribution, create_vm=False, certificate_url=None,
                 execute(setup_foreman_discovery, host=host)
             if os.environ.get('LIBVIRT_KEY_URL') is not None:
                 execute(setup_libvirt_key, host=host)
-            execute(install_puppet_scap_client, host=host)
-            execute(setup_oscap, host=host)
+            if os.environ.get('SATELLITE_VERSION') == '6.1':
```
Comment (rplevka) [inline automation_tools/__init__.py]: perhaps we might change the conditional to include all 6.1+ versions (with 6.2 being released soon) and only exclude 6.0.

**Comment by rplevka** [comment]
ACK pending comment

File: automation_tools/__init__.py (hunk 1 above)
Comment (kbidarkar) [inline automation_tools/__init__.py]: Let's do that when we add support for Sat 6.2 and not now. Currently SATELLITE_VERSION just supports version 6.0 and 6.1

Comment (kbidarkar) [inline automation_tools/__init__.py]: or may be let me change it to `!= '6.0'`

**Comment by sthirugn** [comment]
ACK

**Pull request merged**